root_utils/full_geo_dump.py

```diff
@@ -39,7 +39,7 @@ def geodump(input_file, output_file):
 
 if __name__ == '__main__':
 
-    ROOT.gSystem.Load(os.environ['WCSIMDIR'] + "/libWCSimRoot.so")
+    #ROOT.gSystem.Load(os.environ['WCSIMDIR'] + "/libWCSimRoot.so")
```

Comment:
not sure if this is the best way

Reply:
yeah this line is no longer needed at all with current WCSim. In older WCSim versions it was necessary.